Fixes #10732: Remove attributes from user and use admin for remote ca…

ehelms opened the pull request:
…lls.

jlsherrill commented on app/controllers/katello/dashboard_controller.rb:
```diff
@@ -65,26 +59,9 @@ def subscriptions_totals
     helper_method :quantity
     def quantity
-      current_user.preferences_hash[:dashboard][params[:action]][:page_size] rescue 5
+      5
```
why not use the app-wide page size for this?

ehelms replied:
Good call - will update.

jlsherrill commented on app/models/katello/concerns/user_extensions.rb:
```diff
@@ -58,113 +33,9 @@ def cp_oauth_header
-          else
-            self.preferences_hash[:user][:default_org] = nil
-          end
-        end
```
can we really get rid of default_org?

jlsherrill replied:
ohhh that is in foreman now

jlsherrill commented on app/models/katello/concerns/user_extensions.rb:
```diff
@@ -58,113 +33,9 @@ def cp_oauth_header
-        end
-
-        def subscriptions_match_system_preference
-          self.preferences_hash[:user][:subscriptions_match_system] rescue false
```
any idea what this was supposed to do?

jlsherrill replied:
any of these 'subscriptions_match_*' prefs?

ehelms replied:
I do not recall, I could dig through the old code but these aren't used anywhere that I could find.

jlsherrill commented on db/migrate/20150603045418_remove_user_fields.rb:
```diff
@@ -0,0 +1,18 @@
+class RemoveUserFields < ActiveRecord::Migration
+  def up
+    remove_column :users, :helptips_enabled
```
Bye bye helptips :wave: we hardly knew ye

ehelms commented:
Changed the dashboard controller to use entries_per_page and updated the seeds.rb to remove a requires I missed. This, from my testing, also fixes an issue with katello:reset that was introduced with the backend check work.

jlsherrill commented:
Tested and worked well.  created product, repo, published content view.  Registered a client, subscribed checked bound repos,and checked that applicability generation worked.

ACK
